pkg/virt-handler/isolation/isolation.go

```diff
@@ -403,7 +410,7 @@ func (r *realIsolationResult) Controller() []string {
 }
 
 func (s *socketBasedIsolationDetector) getPid(socket string) (int, error) {
-	sock, err := net.Dial("unix", socket)
+	sock, err := net.DialTimeout("unix", socket, time.Duration(isolationDialTimeout)*time.Second)
```

Comment:
Do you know what the timeout here was before?

Reply:
infinite apparently.